proto/o5/dempe/v1/service.proto

```diff
@@ -44,37 +53,41 @@ message GetMessageRequest {
 
 message CapturedMessage {
   DeadMessage cause = 1;
-
 }
 
 message GetMessageResponse {
   CapturedMessage message = 1;
-  repeated MessageAction actions = 2;
+
+  message GetMessageAction {
+    MessageAction action = 1;
+    o5.auth.v1.Actor actor = 2;
+    google.protobuf.Timestamp timestamp = 3;
+  }
+  repeated GetMessageAction actions = 2;
 }
 
 message MessageAction {
+  string id = 1 [
+    (buf.validate.field).string.uuid = true
+  ];
 
   string note = 2;
 
-  o5.auth.v1.Actor actor = 3;
-
-  google.protobuf.Timestamp timestamp = 4;
-
   oneof action {
     // optional
     ActionDelete delete = 10;
     ActionRequeue requeue = 11;
-    ActionEdit edit = 12;
+    ActionEdit edit = 12; // Will return an error if more than one id is provided
```

Comment:
If a singular message action request is to accept multiple ids of messages to act upon, this is a desirable software side validation. Not particularly a fan of it, but it is an alternative to having two endpoints for performing actions.